src/geometamaker/geometamaker.py

```diff
@@ -418,9 +432,12 @@ def set_band_description(self, band_number, name=None, title=None, abstract=None
             title (str): title for the raster band
             abstract (str): description of the raster band
             units (str): unit of measurement for the band's pixel values
+            type (str): of the band's values, either 'integer' or 'number'
```

Comment:
I don't like using `type` as the argname, since it's a builtin function name, but that is the name of the MCF property, and making the setter's argnames match makes it convenient to do things like mc_a.set_band_description(1, **mc_b.get_band_description(1)).

Reply:
On this note, is it possible to automatically populate the band's min/max in the dimensions section? Dimensions correspond to this feature and seem to be required in the MCF, but not for ISO. I know there is a way to get this information in GDAL.

Reply:
@empavia do you mean the min/max pixel values, or the number of rows and columns?

Reply:
@phargogh the min/max pixels

Reply:
@phargogh actually I believe I am referring to the min/max of the band measurement. For example, the min/max elevation in meters for a DEM. An example is found in this document on Oak: "Z:\global-dataset-cache\awc-isric-soilgrids\awc.tif.ovr.aux.xml"

Reply:
Thanks for bringing this up @empavia , I created a separate issue to address it. Please see #15